tests/utils/pv.go

```diff
@@ -78,6 +80,45 @@ func NewPVDefinition(pvName string, size string, labels map[string]string, stora
 	}
 }
 
+// NewBlockPVDefinition creates a PV definition with volueMode 'Block'
+func NewBlockPVDefinition(pvName string, size string, labels map[string]string) *k8sv1.PersistentVolume {
```

Comment:
Since we have other existing PVs already which are part of the default storageClass, new PVs we create need to be associated with a storage class. kubernetes will not magically determine that a PV matches a PVC if that pvc requests from a different storage class.

So we need the following:
1. Create a storage class for block devices, call it local-block or something. Maybe create it during the cluster-sync
2. Have this PV use the name of that storage class.
3. When we create a PVC that wants to use block volumes, we need to make sure we use that same storage class.

If one of the above doesn't happen, this will not work.

Reply:
The storage class should look like this:
```yaml
apiVersion: v1
items:
- apiVersion: storage.k8s.io/v1
  kind: StorageClass
    name: local-block
    namespace: ""
  reclaimPolicy: Delete
  volumeBindingMode: WaitForFirstConsumer
```
Note this will NOT be a default storage class.

Reply:
done